src/Handler/TagHandler.php

```diff
@@ -88,10 +101,35 @@ static function ($tag) use ($repoPrefix) {
                 },
                 $tags
             );
+
             // Also add a un-prefixed `ez-all` in order to be able to purge all across repos
             $tags[] = 'ez-all';
```

Comment:
We should add this tag at the beginning of the `$tag` array or it will be the first one to go when header length will be reduced.

Reply:
Good point, fixed in b3a5d17 👍